bin/reth/src/args/rpc_server_args.rs

```diff
@@ -179,6 +179,31 @@ pub struct RpcServerArgs {
 }
 
 impl RpcServerArgs {
+    /// Enables the HTTP-RPC server.
+    pub fn with_http(mut self) -> Self {
+        self.http = true;
+        self
+    }
+
+    /// Enables the WS-RPC server.
+    pub fn with_ws(mut self) -> Self {
+        self.ws = true;
+        self
+    }
+
+    /// Change rpc port numbers based on the instance number.
+    ///
+    /// Warning: if `instance` is zero, this will panic.
+    pub fn adjust_instance_ports(&mut self, instance: u16) {
```

Comment:
also panics if instance * 100 would overflow?

Reply:
yes, or if `self.auth_port + instance * 100 - 100` would overflow. This will all wrap in release mode, added more docs on this